### PR TITLE
fletcher:0.5.1

### PR DESCRIPTION
**Pull request by Jollywatt**
I am submitting
- [ ] a new package
- [x] an update for a package

Changes since fletcher v0.5.0:

- Fix nodes which `enclose` absolute coordinates.
- Allow options to `enclose` CeTZ-style coordinate expressions.
- Fix crash with polar coordinates.
- Fix edges which bend at 0deg or 180deg (e.g., `edge("r,r")` or `edge("r,l")`) and enhance the way the corner radius adapts to the bend angle. **Note:** This may change diagram layout from previous versions.
- Improve error messages.
- Add marks for crow's foot notation: `n` (many), `n?` (zero or more), `n!` (one or more), `1` (one), `1?` (zero or one), `1!` (exactly one).

### CODE REVIEW

File: packages/preview/fletcher/0.5.1/docs/gallery/algebra-cube.typ [hunk 1]
```diff
@@ -0,0 +1,47 @@
+#import "@preview/fletcher:0.5.0" as fletcher: diagram, node, edge
```
Comment (elegaanz) [inline packages/preview/fletcher/0.5.1/docs/gallery/algebra-cube.typ:1]: Examples in the gallery still import the previous version, maybe you want to update it here as well.

Comment (Jollywatt) [reply packages/preview/fletcher/0.5.1/docs/gallery/algebra-cube.typ:1]: Ah, thanks. Every time I forget this and you remind me! Will fix...

Comment (jneug) [reply packages/preview/fletcher/0.5.1/docs/gallery/algebra-cube.typ]: I like to use [tbump](https://github.com/your-tools/tbump) for exactly this reason. You configure it once and then upgrade all references to the current version at once.

Comment (Jollywatt) [reply packages/preview/fletcher/0.5.1/docs/gallery/algebra-cube.typ]: All done. And thanks @jneug, I'll definitely use that

**Comment by elegaanz** [comment]
Thank you :)

**Pull request merged**